### DISCUSSION TITLE
傳訊加這2字至少30歲？

### DISCUSSION THREAD

**Discussion by Sylvie0601** [1101-GS4719-C/Python-Community-GS4719-C#1126] [閒聊八卦]
我自己其實平常打字有時候也還是會用XD !
我覺得XD對於電腦打字比較好用，手機如果要打XD還要切換輸入法我認為比較麻煩
所以我之後也都頃向打哈哈、笑死這種來取代XD
不過XD是我的童年ㄟ ! (還是只有我的童年哈哈
小時候都會打XD、:)這種符號，有時候很好笑還要XDDDDD
不知道大家還都會不會用XD呢?
新聞網址 : https://today.line.me/tw/v2/article/kEG5Lx2

**Comment by 12Jesse24** [comment]
會喔，有的時候不知道要回什麼，回了一些敷衍的話後面就會加XD，這樣看起來比較不像在敷衍，或這一些顏文字像是><、^^，聊起天來也比較生動，推薦給大家。

**Comment by mrweixd** [comment]
LINE 應該就是看準我們就想多傳一點文字外的表達才製作貼圖功能吧。像我認識很多人可能在IG上聊一聊就想換賴，因為這樣才能傳貼圖。

**Comment by lazywulf** [comment]
我反而覺得在 LINE 、 Messanger 或簡訊使用句點的比較像是 30+。更多會很像 40 、 50 歲會這樣用。看那樣的訊息會特別像是在讀信件，有點過於嚴肅。

**Comment by eric202501** [comment]
打XD是國小的童年，那個時候yahoo跟ie都還有人在用

**Comment by THH1118** [comment]
這些都是我現在還會用的誒完蛋XD有時候也會用「🤣」這種符號但如果覺得沒有開心到那麼誇張就會用這種文字打XDDD

**Comment by WeiYen1103** [comment]
雖然我現在是沒有甚麼跟比我年紀小的人傳訊息，不知道現在小朋友聊天是用甚麼表情符號，但回想起來XD真的是我小時候就在用的了XDD，我現在回想起來還真的是蠻有趣，每個時代都有不同的流行，但又都會奇特的在某一段時期又流行起幾年前的事物，或許再過幾年也會看到我的小孩大出XD的表情了。

**Comment by Patsy17** [comment]
比起XD我更喜歡用：））
因為這樣就不用切換語法了～
我用這些表情符號都是跟比較熟的人才會用，不然怕被覺得很屁哈哈

**Comment by Yanxxx7979** [comment]
:)同感這個也超好用，頗有一種無奈又白眼的感覺

**Comment by davidteng00000** [comment]
我現在也有用XD欸，難道我老了嗎

**Comment by jim1743** [comment]
以前我在國小的時候，和同學聊天也比較常用XD，但現在聊天比較懶得打這些符號，傳表情貼圖還比較快速，並且更能表達自己現在的心情，再來如果不知道要怎麼回復別人的訊息，我最常用的就是驚訝臉符號，這符號是對我來說是代表我能理解他的想法。

**Comment by EPHMR0524** [comment]
我到現在都還會用XD，實在是懶得點開表情符號(小黃臉)鍵盤發送笑臉給朋友，身邊的朋友也很多都還會用XD，其實用了也不會發出老人的味道，如果是和更熟一點的朋友的話我想表示好笑我會用www，這樣更方便應該只要在同一個鍵位上按三次就好了差滴。

**Comment by cassy0201** [comment]
我覺得還好欸，有時不想打字看起來太兇的時候就會使用，但好像更常用貼圖，甚至可以不用打字超方便!

**Comment by buoioiry** [comment]
很少用XD可是真的不會覺得用這個很老人耶，反而是長輩^^用這種我會被嚇到，感覺很生氣，結果後來問他們都只是覺得這樣看起來比較和善哈哈，算是一種隔閡嗎我朋友們也都覺得這個表情符號看起來很兇，幾乎不會使用到，但現在用XD的朋友我會覺得有點可愛！

**Comment by OHoln** [comment]
對我來說XD就是小時候再玩線上遊戲的時候大家會打的，所以現在看到總有一種違和感，自己也就不太用了

**Comment by 710388** [comment]
感覺是一直打標點符號跟句號那些才會像老人家

**Comment by yubo0oo** [comment]
XD都是宅宅在用的

**Comment by YeeDaniel** [comment]
你是說顏文字嗎 o(*￣︶￣*)o 還有「嗯」「喔」等

**Comment by poiu6fi** [comment]
我自己覺得會加逗號跟句號的人比較像是老人
